staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -473,10 +471,10 @@ func watchHandler(start time.Time,
 	expectedTypeName string,
 	setLastSyncResourceVersion func(string),
 	clock clock.Clock,
-	resourceVersion *string,
 	errc chan error,
 	stopCh <-chan struct{},
 ) error {
+	resourceVersion := ""
```

Comment:
Let's remove and just create this var in every loop below as it was - I think it would be more intuitive.